app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -17,17 +17,52 @@ import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.runtime.staticCompositionLocalOf
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
 import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.ext.settings
+
+/**
+ * Indicates the theme that is displayed.
+ */
+enum class Theme {
+    Light,
+    Dark,
+    Private;
+
+    companion object {
+        /**
+         * Returns the current [Theme] that is displayed.
+         *
+         * @param isPrivate Whether or not private browsing mode is enabled.
+         * @return the current [Theme] that is displayed.
+         */
+        @Composable
+        fun getTheme(isPrivate: Boolean = LocalContext.current.settings().lastKnownMode.isPrivate) =
+            if (isPrivate) {
```

Comment:
I wanted to do `!inComposePreview && LocalContext.current.settings().lastKnownMode.isPrivate` for this conditional but hit a compiler error that looked like https://stackoverflow.com/questions/68375629/andand-condition-should-have-an-if-true-then-false.

Reply:
Filed https://github.com/mozilla-mobile/fenix/issues/24528 for the future.